packages/flutter_tools/lib/src/android/gradle.dart

```diff
@@ -540,6 +540,11 @@ Future<void> _performCodeSizeAnalysis(
   globals.printStatus(
     'A summary of your ${kind.toUpperCase()} analysis can be found at: ${outputFile.path}',
   );
+  globals.printStatus(
+    '\nTo analyze your app size in Dart DevTools, run the following command:\n'
+    'pub global activate devtools; pub global run devtools '
```

Comment:
Can we pub global list to check if devtools is already present and skip the first part if redundant?

Also please add test to analyze_size_test.dart etc.

Reply:
We do want the users to have the latest version of DevTools, though, otherwise the command could fail. 

Here is a related issue: https://github.com/dart-lang/pub/issues/2791. This is another case where it would be nice if `pub global activate` was a no-op if the latest version of a package is already installed. @sigurdm

Reply:
Understood thanks